packer/ansible/roles/aggregate/tasks/main.yml

```diff
@@ -0,0 +1,62 @@
+---
```

Comment:
You only need quotes when there is a {{ variable }}. And for consistency, let's put single quotes around those variables.